### DISCUSSION TITLE
How to use Flowbite in React?

### DISCUSSION THREAD

**Discussion by AringoldX** [Huniko-Team/community#68] [Q&A] [answered]
I am going to use flowbite in react.
If anyone has experience, please introduce me.

**Comment by fhrryDeveloper** [comment, marked as answer]
Flowbite is a UI kit for building responsive and mobile-first websites and web applications using React. It provides a set of pre-designed components and templates that can be easily customized to match the look and feel of a specific project. 
https://www.google.com/search?q=what+is+flowbite+in+react.&rlz=1C1GCEB_enUS1047US1047&oq=what+is+flowbite+in+react.&aqs=chrome..69i57j33i160l2.1472j0j1&sourceid=chrome&ie=UTF-8
